app/models/film.rb

```diff
@@ -16,4 +17,8 @@ class Film < ApplicationRecord
   validates :name, :movie, :trailer, :image, presence: true
 
   scope :create_at_desc, -> {order created_at: :desc}
+  scope :action_films, -> {where category_id: 1}
```

Comment:
dùng enum đi em

Reply:
vâng ạ

Reply:
@phamhuuluan ko thêm mới được thì lỗi nó hiện như nào em

Reply:
'1' is not a valid category_id ạ

Reply:
e dùng enum đi. còn khi create/update e phải dùng text của enum update ms dc. vd:
enum status: {active: 1, inactive: 0}
thì khi create/update sẽ là user.update_attributes status: :active
e đọc kĩ cách sử dụng enum nhé

Reply:
dạ vâng ạ , e đã sửa r ạ